src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Telemetry/TelemetryScope.cs

```diff
@@ -4,10 +4,12 @@
 using System;
 using System.Diagnostics;
 using Microsoft.AspNetCore.Razor.PooledObjects;
+using Microsoft.AspNetCore.Razor.Utilities;
 
 namespace Microsoft.AspNetCore.Razor.Telemetry;
 
-internal sealed class TelemetryScope : IDisposable
+[NonCopyable]
+internal struct TelemetryScope : IDisposable
 {
     public static readonly TelemetryScope Null = new();
```

Comment:
Now that `TelemetryScope` is a struct, this is an unnecessary affordance. Plus, there's no such thing as "Null" for a struct. :smile: Please remove these and update all usages to just refer to `default`.